src/js/components/Autocomplete.js

```diff
@@ -242,7 +243,7 @@ class Autocomplete extends Component {
 
     return (
       <Fragment>
-        {suggestionsListComponent}
+        {!isChatBot && suggestionsListComponent}
```

Comment:
In case the condition is false, it will return the empty Fragment? Is that expected?

Reply:
Thank you ji, this should be fixed with recent commit.

Reply:
I reverted this back, react gives an error if we don't return that empty fragment.